packages/cli/src/services/__tests__/workflow-statistics.service.test.ts

```diff
@@ -187,6 +198,7 @@ describe('WorkflowStatisticsService', () => {
 		});
 
 		test('should only create metrics for production successes', async () => {
+			mockDBCall();
```

Comment:
This `mockDBCall` setup is rather opaque. Comes from before though.

Reply:
Happy to inline it. I don't like custom test hellpers defined in the same test file either. They make it harder to read the tests in one go.

I'll do that.

Reply:
Actually, the function branches on 3 different db implementations, so I'd rather not inline this into every test. 

I'm not sure why this test mocks and asserts the entity manager instead of just the repository? It's basically duplicating the repository test. 🤔 

WDYT about removing these tests alltogether?

Reply:
Wow if they're duplicates then definitely 👍🏻

Reply:
Actually there were no repository tests at all.

So I removed the mocks so that the service tests test the SQL in the repository as well. I also removed some duplication and improved the test names so it's clearer what they test.